test/lib/linter/amdTranspiler/snapshots/transpiler.ts.md

```diff
@@ -485,6 +536,9 @@ Generated by [AVA](https://avajs.dev).
     import Button from "sap/m/Button";␊
     const someValue1 = "foo";␊
     const someValue2 = new Button();␊
+    /**␊
+     * @namespace com.ui5.troublesome.app.controller␊
+     */␊
     export default class BaseController_1 extends Controller {␊
```

Comment:
The idea to reconstruct the original class name from the `@namespace` + the local class name won't work in this case. We can't guarantee that the correct name will be used as there might be conflicts with existing variable names.

Reply:
That's true, we can't! 
And further more, the definition in the XMLView then would never match.

I was thinking the following: the filename contains the actual (referenced by the XMLView) className. So, we can merge the namespace along with the cleared filename. For example:
Main.view.xml
```xml
controllerName="com.ui5.troublesome.app.controller.Main"
```
then the corresponding controller would be:
-> Main.controller.xml

What do you think? Any other ideas?